app/views/shared/check_answers/_merits.html.erb

```diff
@@ -97,11 +97,11 @@
       ) %>
 </dl>
 
-<div class='govuk-body'><%= simple_format statement_of_case.statement %></div>
+<div class='govuk-body' id="statement-of-case"><%= simple_format statement_of_case.statement %></div>
 
 <%= render 'shared/check_answers/section_break' if respondent.bail_conditions_set %>
 
-<div class="govuk-summary-list govuk-!-margin-bottom-2">
+<dl class="govuk-summary-list govuk-!-margin-bottom-2">
```

Comment:
There has been some inconsistency as to whether `div` or `dl` tags should be used to define a "govuk-summary-list". To remove this inconsistency, there was a PR that went through and changed them to use `div` - so at least there was some consistency - especially as some of them contained inner elements that were constructed dynamically. This seems to reversing that change.

If we should be using `dl` tags for all `govuk-summary-list` lists, then that change needs to be done throughout the code base, and not just in isolated places like this.

Reply:
In this instance, the opeining was a `div` and the closing was a `dl`.  

We should be using the `dl` `dt` `dd` structure as per the design system.  I went through changing them all to this structure as per the Webhint tickets.  I thought I was going mad seeing how many of them I seemed to have missed.  What was this PR?

Reply:
It was this PR: https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/705

Looks like I jumped the wrong way. - should have made them all `dl`.

Reply:
I'll raise another ticket to get them moved back.

Reply:
Can you please reference #705 in the new ticket. I'll look out for it so I can correct my mistake.

Reply:
Done (AP-924)  Thanks.